pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1354,45 +1354,6 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(causes).To(HaveLen(1))
 			Expect(causes[0].Field).To(Equal("fake.networks[0]"))
 		})
-		It("should reject networks with a multus network source and slirp interface", func() {
-			enableSlirpInterface()
```

Comment:
I think this method should now be removed from the file, isn't it?
I found some usages of it, but it is on tests checking ports. It seems other binding can be used instead.

Reply:
I prefer to remove it once I move everything out and not now. It reduces the number of changes needed.